docs/proxysql-conf.md

```diff
@@ -137,201 +108,221 @@ proxysql:
+To address this limitation, the Operator is integrated with the [`pxc_scheduler_handler` :octicons-link-external-16:](https://docs.percona.com/proxysql/psh-overview.html) tool starting with version 1.19.0. This external ProxySQL scheduler ensures the read/write splitting is distributed as follows:
 
-``` {.bash data-prompt="$" }
-$ kubectl create -f deploy/my-proxysql-secret.yaml
-```
+* **SELECT queries** (without `FOR UPDATE`) are sent evenly to all PXC nodes or to all nodes except the primary, depending on your configuration
+* **Non-SELECT queries** and **SELECT FOR UPDATE** queries are sent to the primary node
+* The scheduler automatically manages the primary node, ensuring only one primary exists at a time
 
-!!! note
+As a result, you achieve:
 
-    Do not forget to restart Percona XtraDB Cluster to ensure the
-    cluster has updated the configuration.
+* Better performance through faster query processing and increased throughput
+* Higher reliability by preventing single-node bottlenecks and points of failure
+* Healthier cluster through early detection of replication lag and node issues
+* Efficient resource utilization
+* Improved user experience with consistent, predictable response times
 
-## Accessing the ProxySQL Admin Interface
+### Enable the scheduler
 
-You can use [ProxySQL admin interface :octicons-link-external-16:](https://www.percona.com/blog/2017/06/07/proxysql-admin-interface-not-typical-mysql-server/) to  configure its settings.
+The scheduler is disabled by default to maintain backward compatibility. You can enable it by setting `proxysql.scheduler.enabled=true` in your Custom Resource.
 
-Configuring ProxySQL in this way means connecting to it using the MySQL
-protocol, and two things are needed to do it:
+1. Edit the `deploy/cr.yaml` file and add the scheduler configuration:
 
-* the ProxySQL Pod name
+    ```yaml
+    proxysql:
+      enabled: true
+      size: 3
+      image: percona/percona-xtradb-cluster-operator:{{ release }}-proxysql
+      scheduler:
+        enabled: true
+    ```
+
+2. Apply the configuration:
 
-* the ProxySQL admin password
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cr.yaml -n <namespace>
+    ```
 
-You can find out ProxySQL Pod name with the `kubectl get pods` command,
-which will have the following output:
+When the scheduler is enabled, you should see:
 
-```default
-$ kubectl get pods
-NAME                                              READY   STATUS    RESTARTS   AGE
-cluster1-pxc-node-0                               1/1     Running   0          5m
-cluster1-pxc-node-1                               1/1     Running   0          4m
-cluster1-pxc-node-2                               1/1     Running   0          2m
-cluster1-proxysql-0                               1/1     Running   0          5m
-percona-xtradb-cluster-operator-dc67778fd-qtspz   1/1     Running   0          6m
-```
+* **Hostgroup 10** (readers): All PXC nodes with equal or weighted distribution
```

Comment:
We add nodes with weighted distribution when writer (pxc-0) has the lowest priority + 8XXX Groups  (although maybe we do not need to mention 8XXX)

```
10	some-name-pxc-0	600	ONLINE
10	some-name-pxc-1	999	ONLINE
10	some-name-pxc-2	998	ONLINE
11	some-name-pxc-0	1000000	ONLINE
8010	some-name-pxc-0	600	ONLINE
8010	some-name-pxc-1	999	ONLINE
8010	some-name-pxc-2	998	ONLINE
8011	some-name-pxc-0	1000000	ONLINE
8011	some-name-pxc-1	999	ONLINE
8011	some-name-pxc-2	998	ONLINE
```